docker-compose.yml

```diff
@@ -1,6 +1,19 @@
 version: '3.1'
 services:
 
+    mariadb_oauth:
+        image: 'mariadb:10.2'
+        working_dir: /application
+        volumes:
+            - '.:/application'
+        environment:
+            - MYSQL_ROOT_PASSWORD=goteo
+            - MYSQL_DATABASE=goteo_oauth
+            - MYSQL_USER=goteo
+            - MYSQL_PASSWORD=goteo
+        ports:
+            - '12003:3306'
+
```

Comment:
This way we don't have to manually create the OAuth database. A migration is executed when starting Docker compose with the OAuth tables creation.